packages/mui-base/src/utils/useAnchorPositioning.ts

```diff
@@ -86,9 +95,14 @@ export function useAnchorPositioning(
     arrowPadding = 5,
     mounted = true,
     trackAnchor = true,
+    allowAxisFlip = true,
+    open,
     nodeId,
+    inner: innerMiddleware,
+    innerOptions = {},
   } = params;
 
+  const standardMode = innerOptions.touchModality || !(!innerOptions.fallback && innerMiddleware);
```

Comment:
IMO this deserves a comment explaining what it is and what's the other mode.

Reply:
`alignPopupToTrigger` (the inverse of `alignOptionToTrigger`) should now be the name of this